src/test/java/teammates/common/datatransfer/UserInfoCookieTest.java

```diff
@@ -0,0 +1,26 @@
+package teammates.common.datatransfer;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+import org.testng.annotations.Test;
+
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@link UserInfoCookie}.
+ */
+public class UserInfoCookieTest extends BaseTestCase {
+    private UserInfoCookie uc = new UserInfoCookie("MockId");
+
+    @Test
+    public void testIsValid() {
```

Comment:
Now that we're adding this test, let's add the missing test for the first part of the validity check: `StringHelper.isCorrectSignature(userId, verificationCode)`